app/services/sap_session_api.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+#
+# Uses Suara's SAP API to check if the user is valid.
+# To send a request you MUST provide:
+# - username: the username of the user
+#
+class SapSessionApi
+  WSDL_URL = ENV.fetch('SAP_API_WSDL_URL') { '' }
+  SUPPORT_ACCOUNT = 'sergiurzayg.ext.coditramuntana'
```

Comment:
Could you use an environment variable instead, please?

Reply:
Done!